_data/library-tools.yml

```diff
@@ -24,16 +24,16 @@
   description: |
     A command-line tool that makes it easy to encrypt and decrypt data using Amazon Key Management Service (KMS).
 
-- name: "ssh-iam"
+- name: "ssh-grunt"
   tags:
     - AWS
   icons:
     - title: Terraform
       image: icon-terraform.png
     - title: Go
       image: icon-go.png
-  docs_url: "https://github.com/gruntwork-io/module-security-public/tree/master/modules/ssh-iam"
-  full_url: "https://github.com/gruntwork-io/module-security/tree/master/modules/ssh-iam"
+  docs_url: "https://github.com/gruntwork-io/module-security-public/tree/master/modules/ssh-grunt"
+  full_url: "https://github.com/gruntwork-io/module-security/tree/master/modules/ssh-grunt"
   description: |
     A tool that allows you to manage SSH access to EC2 Instances using AWS IAM. Developers can use their personal SSH
```

Comment:
Can you update this to say "manage SSH access to EC2 Instances using either AWS IAM or your Identity Provider (e.g., ADFS, Google, Okta, etc).